docs/components/TemplatesSystem/features/twig_nodes.rst

```diff
@@ -78,7 +79,7 @@ Here is commented example of pagination:
                 * lastPage (int) : represents the total number of existing pages
                 * showAlwaysFirstAndLast (bool) : Always show first and last link (just disabled)
             #}
-            {% include '@SWPTemplatesSystem/pagination.html.twig' with {
+            {% include 'partials/pagination.html.twig' with {
```

Comment:
Hey, 

what's wrong with default one provided by TemplatesSystem bundle? Maybe we should replace it with one paseted bellow?